src/resources/elements/timeLeft/timeLeft.scss

```diff
@@ -7,18 +7,29 @@ $comingUpColor: $Orange;
 $comingUpBackground: #3e3038;
 $farOffColor: #cecece;
 $farOffBackground: #30343e;
-$liveColor: $Green;
+
+$completedColor: #18bab0;
```

Comment:
TO DO: These need to come from the color palette

Reply:
Moved the variables to the design system colors file. This still might change once @v0ort will update the palette in the Figma file.